kubernetes/linux/Dockerfile.multiarch

```diff
@@ -22,7 +22,7 @@ ENV tmpdir /opt
 RUN tdnf clean all
 RUN tdnf repolist --refresh
 RUN tdnf -y update
-RUN tdnf install -y build-essential wget openssl curl sudo net-tools cronie rsyslog dmidecode gnupg make logrotate busybox gawk tar && rm -rf /var/lib/apt/lists/*

```

Comment:
does this mean openssl components are already present in the distroless image?

Reply:
yes, openssl already available in distroless. Mariner team recommends to use the version in distroless instead of copying binaries from base to distroless.